misc/scripts/install-local.sh

```diff
@@ -198,10 +200,15 @@ function makeVirtualDeb {
 			unset z
 			# tab over the next line
 			echo -ne "\t"
-			ask "Do you want to install the optional dependencies" Y
-			if [[ $answer -eq 1 ]]; then
-				for optdep in "${optdeps[@]}"; do
-					deps+=" ${optdep%%: *}"
+			select_options "Select optional dependencies to install" "${#optdeps[@]}"
+			choices=( $(cat /tmp/pacstall-select-options) )
+			if [[ "${choices[0]}" != "n" ]]; then
+				for i in "${choices[@]}"; do
+					(( i-- ))
+					s="${optdeps[$i]}"
+					if [[ -n $s ]]; then
```

Comment:
Could this just use `"${#s}"` or something so the variable can be quoted? This probably ain't gonna cause issues regardless, but I think Bash best practices should still be followed.

Reply:
I believe variables inside `[[` don't need to be quoted.

Reply:
They probably don't *need* to, but I still think it'd be best if they were still

Reply:
Our policy at pacstall is don't quote variables inside `[[ ]]` unless they somehow need to.

Reply:
Fair enough